api/src/main/java/org/openmrs/module/eptsreports/reporting/cohort/evaluator/ResumoMensalTransferredOutCohortDefinitionEvaluator.java

```diff
@@ -45,7 +45,7 @@ public EvaluatedCohort evaluate(CohortDefinition cohortDefinition, EvaluationCon
     q.append("FROM (SELECT patient_id, ");
     q.append("             Max(transferout_date) transferout_date ");
     q.append("      FROM (SELECT p.patient_id, ");
-    q.append("                   ps.start_date transferout_date ");
+    q.append("                   Max(ps.start_date) AS transferout_date ");
```

Comment:
@cfaife guess there's a need to put group by

Reply:
done